src/Intervals.jl

```diff
@@ -42,6 +42,9 @@ export Bound,
        AbstractInterval,
        Interval,
        AnchoredInterval,
+       LeftEndpoint,
+       RightEndpoint,
+       AnchorEndpoint,
```

Comment:
I'm less certain about exporting these. Maybe we should just use symbols `:left`, `:right`, and `:anchor` for the initial version of rounding

Reply:
The deprecations mean they were being exported anyway (except for AnchorEndpoint)